aztec/src/main/kotlin/org/wordpress/aztec/toolbar/AztecToolbar.kt

```diff
@@ -371,9 +394,36 @@ class AztecToolbar : FrameLayout, OnMenuItemClickListener {
         }
     }
 
+    fun getHeadingMenu(): PopupMenu? {
+        return headingMenu
+    }
+
+    fun getListMenu(): PopupMenu? {
+        return listMenu
+    }
+
+    fun getSelectedHeadingMenuItem(): TextFormat? {
+        if (headingMenu?.menu?.getItem(0)?.isChecked!!) return TextFormat.FORMAT_PARAGRAPH
```

Comment:
I don't think we should be using number literals (the same applies to the list menu). By looking at `headingMenu?.menu?.getItem(0)` there is no way to tell what menu item we're trying to get without a including comment. Also, if the order changed in the resource file, it might break this. With `findItem()` you can specify an id to get a particular item.